runtime/include/blueoil.hpp

```diff
@@ -23,9 +23,17 @@ extern "C" {
 
 
 namespace blueoil {
-struct Tensor {
+class Tensor {
+public:
   std::vector<float> data;
   std::vector<int> shape;
+  int elems();
+  static Tensor zeros(std::vector<int> shape);
+  static Tensor array(std::vector<int> shape, float *data);
```

Comment:
why it is not a constructor?

Reply:
zeros, array imitates the NumPy API.
But, i agree it shall be constructor.

https://github.com/blue-oil/blueoil/pull/9/commits/8f5da4e01b17a27549d1e55ecd4faedc259a7b91
https://github.com/blue-oil/blueoil/pull/9/commits/37fcc6e4ea99cea025187e020fad1040a8fb458d